en_US/data-integration/data-bridge-iotdb.md

```diff
@@ -1,37 +1,48 @@
 
-- Knowledge about EMQX data integration [rules](./rules.md)
+This page provides a comprehensive introduction to the data integration between EMQX and Apache IoTDB with practical instructions on creating and validating the data integration.
 
-- Knowledge about [data bridges](./data-bridges.md)
+## How It Works
 
-- Basic knowledge of UNIX terminal and commands
+<!-- In this section, you explain how the data integration works together with the rule engine by introducing the architecture, emphasizing the simplicity and no coding required….You can also combine the working principle with a real-life use case. Some typical scenarios include connected vehicles, IIoT, power and energy, etc. You can describe where the data will be used eventually and what value they can bring to the business under the specific scenario. Use sequential steps to describe how data flows from devices to the data integration, and then to the data storage…-->
+
+The Apache IoTDB data integration is an out-of-the-box feature in EMQX designed to bridge the gap between raw MQTT-based time series data and IoTDB's powerful data storage capabilities. With a built-in [rule engine](https://docs.emqx.com/en/enterprise/v5.1/data-integration/rules.html) component, the integration simplifies the process of ingesting data from EMQX to IoTDB for storage and query, eliminating the need for complex coding.
+
+The diagram below illustrates a typical architecture of data integration between EMQX and IoTDB. <!-- This image needs to be modified to be IoTDB specific-->
+
+<img src="./assets/IoTDB_bridge_architecture.png" alt="IoTDB_bridge_architecture" style="zoom:67%;" />
 
-:::
+The workflow of the data integration is as follows:
 
-## Feature List
+1. **Message publication from IoT devices**: Devices, whether they are part of connected vehicles, IIoT systems, or energy management platforms, send messages via MQTT based on their operational states, readings, or triggered events.
+2. **Message reception by EMQX:** As an MQTT Broker, EMQX receives these MQTT messages from the IoT devices. It serves as the centralized hub for handling MQTT-based communication.
+3. **Message data processing:** When a message arrives, it passes through the rule engine and is then processed by the rule defined in EMQX. The rules, based on predefined criteria, determine which messages need to be routed to IoTDB. If any rules specify payload transformations, those transformations are applied, such as converting data formats, filtering out specific information, or enriching the payload with additional context.
+4. **Data buffering**: EMQX provides an on-disk message buffer to prevent data loss when the IoTDB is unavailable. Data is temporarily held in the buffer.
```

Comment:
```suggestion
4. **Data buffering**: EMQX provides an in-memory message buffer to prevent data loss when the IoTDB is unavailable. Data is temporarily held in the buffer, and may be offloaded to disk to prevent memory overload. Note that data is not preserved if the data integration or the EMQX node is restarted.
```